storm-core/src/clj/org/apache/storm/daemon/executor.clj

```diff
@@ -280,7 +281,7 @@
 (defn- mk-disruptor-backpressure-handler [executor-data]
   "make a handler for the executor's receive disruptor queue to
   check highWaterMark and lowWaterMark for backpressure"
-  (disruptor/disruptor-backpressure-handler
+  (DisruptorUtils/disruptorBackpressureHandler
```

Comment:
Could we just inline the impl here?  There isn't much point in having this translated to java when it is still so dependent on clojure.